app/src/main/res/values-sk/strings.xml

```diff
@@ -11,5 +11,10 @@
     <string name="stats_filter_from">Od</string>
     <string name="stats_filter_to">Do</string>
     <string name="stats_filter_button">Filtrovať</string>
+    <string name="button_for_adding_a_new_spending">Pridaj nový výdavok</string>
+    <string name="back_button">Tlačidlo späť</string>
```

Comment:
Stačí aby bolo len "Späť" či ?

Reply:
No, explicitne som napísal "Tlačidlo", pretože tento string sa využíva v properte `contentDescription`, ktorá sa využíva pre accessibility, teda lepšie mi prišlo, že ak si človek nejakým čítačom prejde cez tlačidlo, tak mu nepovie len "Späť", ale "Tlačidlo späť". Ale možno by bolo lepšie, pre konzistenciu, napísať iba "Spät", pretože aj string nad tým má hodnotu "Pridaj nový výdavok" a nie "Tlačidlo pridaj nový výdavok". Zmením to.